rootfs-configs.yaml

```diff
@@ -107,3 +107,14 @@ rootfs_configs:
       - libext2fs2
     script: "scripts/buster-cros-ec-tests.sh"
     test_overlay: ""
+
+  sid:
+    rootfs_type: debos
+    debian_release: sid
+    debian_mirror: http://deb.debian.org/debian-ports
+    keyring_package: debian-ports-archive-keyring
+    keyring_file: /usr/share/keyrings/debian-ports-archive-keyring.gpg
+    arch_list:
+      - riscv64
+    extra_packages_remove: *extra_packages_remove_buster
```

Comment:
If we need to remove a different list of packages for `sid` then please make a copy of the list used for `buster` here and adjust it accordingly.  It's likely that this will keep evolving due to the nature of `sid`.  When `bullseye` gets released we'll bump all the rootfs variants to use it and replace the list of packages to remove.

Reply:
OK, will update with new list for sid.